tests/server/blueprints/cases/test_cases_views.py

```diff
@@ -17,7 +17,7 @@ def test_parse_raw_gene_symbols(app):
     """ Test parse gene symbols"""
 
     # GIVEN a list of autocompleted gene symbols
-    gene_symbols = ["MUTYH |POT1", "POT1| APC |PMS2"]
+    gene_symbols = ["MUTYH |POT1", "POT1 0.1|APC|PMS2"]
 
     # WHEN converting to hgnc_ids
     hgnc_symbols = parse_raw_gene_symbols(gene_symbols)
```

Comment:
Something I noticed but it was already there before this PR:
a couple of lines after this, there is this code:

```
# THEN the appropriate set of hgnc_ids should be returned
    assert hgnc_symbols == {"APC", "MUTYH", "PMS2", "POT1"}
```

Could you change, the comment, `hgnc_ids` to gene symbols?